examples/formbot/actions/actions.py

```diff
@@ -1,140 +1,103 @@
 from typing import Dict, Text, Any, List, Union
 
-from rasa_sdk import Tracker
+from rasa_sdk import Action, Tracker
 from rasa_sdk.executor import CollectingDispatcher
 from rasa_sdk.forms import FormAction
+from rasa_sdk.events import (
+    SlotSet,
+    EventType,
+)
 
+class ValidateRestaurantForm(Action):
+    def name(self) -> Text:
+        return "validate_restaurant_form"
+
+    async def run(self, dispatcher, tracker, domain) -> List[EventType]:
```

Comment:
```suggestion
    async def run(self, dispatcher: CollectingDispatcher, tracker: Tracker, domain: Domain) -> List[EventType]:
```

I think `Domain` needs to be imported